src/constant/index.js

```diff
@@ -83,3 +91,20 @@ export const LIKECOIN_NFT_CAMPAIGN_ITEMS = IS_TESTNET
       'likenft1c86xz87908m90pl7c6ygulyaxcccsdgy9f7rutxg07xe24wvuksswzukgw',
       'likenft1qmhrjp4a72w8g72uw64g64g4kmj2gyrje50vg9eus46hn79s7acsk8zpwm',
     ];
+
+export const LIKECOIN_WALLET_CONNECTOR_CONFIG = {
+  chainId: LIKECOIN_CHAIN_ID,
+  chainName: IS_TESTNET ? 'LikeCoin public test chain' : 'LikeCoin chain',
+  rpcURL: LIKECOIN_CHAIN_NFT_RPC,
+  restURL: LIKECOIN_CHAIN_NFT_LCD,
+  coinType: 118,
+  coinDenom: LIKECOIN_CHAIN_DENOM,
+  coinMinimalDenom: LIKECOIN_CHAIN_MIN_DENOM,
+  coinDecimals: 9,
+  bech32PrefixAccAddr: 'like',
+  bech32PrefixAccPub: 'likepub',
+  bech32PrefixValAddr: 'likevaloper',
+  bech32PrefixValPub: 'likevaloperpub',
+  bech32PrefixConsAddr: 'likevalcons',
+  bech32PrefixConsPub: 'likevalconspub',
+};
```

Comment:
Just use a separated `network.js` so that we can just reuse keplr configs?

Reply:
sure